ext/date/php_date.stub.php

```diff
@@ -375,9 +375,7 @@ public static function getLastErrors(): array|false {}
      */
     public function format(string $format): string {}
 
-    /**
-     * @tentative-return-type
-     */
+    /** @tentative-return-type */
```

Comment:
Please don't make unnecessary formatting changes.

Reply:
Actually, IMO this is an useful change, makes the stub much shorter and easier to overview. Actually, I asked the same in different extensions.

Reply:
They shouldn't be in the same commit (which it will end up in after squashing), and hence, shouldn't be part of this PR to begin with.